src/ucp/core/ucp_worker.c

```diff
@@ -2665,9 +2683,21 @@ void ucp_worker_destroy(ucp_worker_h worker)
 
     UCS_ASYNC_BLOCK(&worker->async);
     uct_worker_progress_unregister_safe(worker->uct, &worker->keepalive.cb_id);
+
+    /* Destroying UCP EP in ucp_ep_disconnected() could start UCT EP discarding
+     * operations.
+     * Do cleanup of discarding functionality two times:
+     * - before destroying of all UCP EP in order to complete destroying of all
+     *   remaining UCP EPs (they are destroyed during completion of discarding
```

Comment:
something is wrong in this sentence. before destroying all ucp eps, to complete destroying of UCP eps